### ISSUE TITLE
Cannot create a replacement default file browser with different aria label, icon, etc.

### ISSUE THREAD

**Issue by afshin** [jupyterlab/jupyterlab#15689]
The default file browser's title, aria label, and role are set outside of the plugin that `provides` the default file browser (`@jupyterlab/filebrowser-extension:default-file-browser`), and are instead defined in the extension that adds the default file browser to the lab shell, here:

https://github.com/jupyterlab/jupyterlab/blob/ae54125a155946e6aa817bc9d8ea21cd0b497631/packages/filebrowser-extension/src/index.ts#L437-L458

Arguably, these attributes are inherently part of the "content" of the file browser and the responsibility for setting them and keeping them updated should be with the `@jupyterlab/filebrowser-extension:default-file-browser` plugin instead so that an alternative file browser (e.g., suppose it is called "Git Browser" or "Cloud Browser", has a different icon, etc.) can correctly display its icon, caption, aria label, etc. without it being clobbered when it is added to JupyterLab.

** Proposal **
Move this functionality from `@jupyterlab/filebrowser-extension:widget` to `@jupyterlab/filebrowser-extension:default-file-browser`.

cc: @DenisaCG
